Jenkinsfile

```diff
@@ -0,0 +1,166 @@
+                    else
+                        echo "Skipping Python SDK"
+                    fi
+
+                    if [ "$BUILD_SDK_JAVASCRIPT" = true ] ; then
+                        echo "Building JavaScript SDK"
+                        export BUILT_SDKS="$BUILT_SDKS, JavaScript"
+                        node sdkBuilder.js \
+                        --config $WORKSPACE/resources/sdk/purecloudjavascript/config.json \
+                        --localconfig $WORKSPACE/platform-client-sdk-internal-config/prod/javascript/prodConfig.yml
+                    else
+                        echo "Skipping JavaScript SDK"
+                    fi
+
+                    if [ "$BUILD_SDK_IOS" = true ] ; then
+                        echo "Building iOS SDK"
+                        export BUILT_SDKS="$BUILT_SDKS, iOS"
+                        node sdkBuilder.js \
+                        --config $WORKSPACE/resources/sdk/purecloudios/config.json \
+                        --localconfig $WORKSPACE/platform-client-sdk-internal-config/prod/ios/prodConfig.yml
+                    else
+                        echo "Skipping iOS SDK"
+                    fi
+
+                    if [ "$BUILD_GUEST_CHAT_SDK_JAVA" = true ] ; then
+                        echo "Building Guest Chat Java SDK"
+                        export BUILT_SDKS="$BUILT_SDKS, Java-guest"
+                        node sdkBuilder.js \
+                        --config $WORKSPACE/resources/sdk/purecloudjava-guest/config.json \
+                        --localconfig $WORKSPACE/platform-client-sdk-internal-config/prod/java-guest/prodConfig.yml
+                    else
+                        echo "Skipping Guest Chat Java SDK"
+                    fi
+
+                    if [ "$BUILD_GUEST_CHAT_SDK_JAVASCRIPT" = true ] ; then
+                        echo "Building Guest Chat JavaScript SDK"
+                        export BUILT_SDKS="$BUILT_SDKS, JavaScript-guest"
+                        node sdkBuilder.js \
+                        --config $WORKSPACE/resources/sdk/purecloudjavascript-guest/config.json \
+                        --localconfig $WORKSPACE/platform-client-sdk-internal-config/prod/javascript-guest/prodConfig.yml
+                    else
+                        echo "Skipping Guest Chat JavaScript SDK"
+                    fi
+
+                    if [ "$BUILD_SDK_GO" = true ] ; then
+                        echo "Building GO SDK"
+                        export BUILT_SDKS="$BUILT_SDKS, GO"
+                        node sdkBuilder.js \
+                        --config $WORKSPACE/resources/sdk/purecloudgo/config.json \
+                        --localconfig $WORKSPACE/platform-client-sdk-internal-config/prod/go/prodConfig.yml
+                    else
+                        echo "Skipping Go SDK"
+                    fi
+
+                    if [ "$BUILD_WEB_MESSAGING_SDK_JAVA" = true ] ; then
```

Comment:
This can be removed for now. We might not be building the web messaging SDK as part of this job

Reply:
Removed